Engine/Engine.cpp

```diff
@@ -638,8 +651,8 @@ Engine::Engine() {
     // pKeyboardInstance = new Keyboard;
     // pGammaController = new GammaController;
 
-    keyboardInputHandler = ::keyboardInputHandler;
-    keyboardActionMapping = ::keyboardActionMapping;
+    keyboardInputHandler = nullptr;
```

Comment:
Is this intended?

Reply:
Yes, they are now set [https://github.com/CkNoSFeRaTU/world-of-might-and-magic/blob/0170a3c76bbba446f4d1293338567db9e425d76b/src/Application/Game.cpp#L202](here). Engine are now loading first before creating window, to read configuration and parse startup parameters. And keyboardinputshandler is depending on window in constructor. So it was done that way.